packages/react-app/package.json

```diff
@@ -32,11 +32,14 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.3.5",
+    "connected-react-router": "^6.8.0",
+    "history": "^5.0.0",
```

Comment:
I am getting an error when I try to navigate.
I looked into it a bit and found this:
https://github.com/supasate/connected-react-router/issues/312
basically setting history version to:
"history": "^4.10.1",
solved it for me

Reply:
Hi, 
It worked for me, but I am still facing one more issue, 
When I am manually changing back/forward in browser , history and history.listen is working fine.

when I am changing route by react-router, locationChange is not getting triggered,

"history": "4.10.1",
 "connected-react-router": "^6.8.0",
   "react-router": "^5.2.0",